src/org/labkey/test/params/experiment/SampleSetDefinition.java

```diff
@@ -0,0 +1,94 @@
+package org.labkey.test.params.experiment;
```

Comment:
Completely unrelated to this code...
Should we add a read-me to the params folder to help clarify what is appropriate to add here?

Reply:
It should definitely be documented but we can add it to our existing documentation or maybe add a `README` at the repository root